Gemfile

```
@@ -74,3 +73,5 @@ gem 'popper_js'
 gem 'twitter-typeahead-rails', '0.11.1.pre.corejavascript'
 gem 'jquery-rails'
 gem 'auto-linker'
+
+gem "funnel_cake_index", github: "tulibraries/funnel_cake_index", branch: "FUNCAKE-104a-Missing_dependencies"
```

Don't need the branch anymore

```suggestion
gem "funnel_cake_index", github: "tulibraries/funnel_cake_index"
```